airflow/www/static/js/dag/details/Dag.tsx

```diff
@@ -72,8 +122,13 @@ const Dag = () => {
         <Tr key={val}>
           <Td>
             <Flex alignItems="center">
-              <SimpleStatus state={val as TaskState} mr={2} />
-              <Text>Total {val}</Text>
+              <SimpleStatus state={val as TaskState} />
+              <LinkButton
+                href={getRedirectUri(val)}
```

Comment:
We probably want this to be just a link, not a button

Reply:
> A few small changes to maintain UX compatibility for now.
> 
> A rebase is needed
> 
> Looks like we're not parsing objects correctly: <img alt="Screenshot 2023-06-02 at 2 16 00 PM" width="439" src="https://user-images.githubusercontent.com/4600967/242961167-0039f81d-a945-4a06-8f60-78c9d37a719a.png">
> 
> Final big piece of feedback, (which can be a secondary PR). It's a very long list of data, I think we should play with other ways to show this data. At least two columns on wide screens. Or maybe a dashboard style with various cards?

Made few changes. Can you recheck, its should have gone.

Reply:
Made it a Link! 
Shall we direct the users to the Link in a new TAB?

Reply:
I think the same tab is fine, new tab is mainly for external links. Wanna make it blue and/or underlined to let users still know they can click it?

Reply:
Done